src/pushprocessor.js

```diff
@@ -268,7 +268,7 @@ function PushProcessor(client) {
         }
 
         const val = valueForDottedKey(cond.key, ev);
-        if (!val || typeof val != 'string') {
+        if (isNullOrUndefined(val) || typeof val != 'string') {
```

Comment:
Is the left side needed here? `typeof null === "object"` and `typeof undefined === "undefined"`...

Fine to keep if it's clearer as-is though. I guess a comment on what cases we mean to include vs. exclude would help.

Reply:
Probably not, but I'm terrified of changes in this area of code (given age).